appgate/__main__.py

```diff
@@ -87,9 +87,9 @@ def get_context(args: OperatorArguments,
     device_id = os.getenv(DEVICE_ID_ENV) or args.device_id
     controller = os.getenv(HOST_ENV) or args.host
     timeout = os.getenv(TIMEOUT_ENV) or args.timeout
-    two_way_sync = os.getenv(TWO_WAY_SYNC_ENV) or ('1' if args.two_way_sync else '0')
-    dry_run_mode = os.getenv(DRY_RUN_ENV) or ('1' if args.dry_run else '0')
-    cleanup_mode = os.getenv(CLEANUP_ENV) or ('1' if args.cleanup else '0')
+    two_way_sync = (os.getenv(TWO_WAY_SYNC_ENV) or ('0' if args.no_two_way_sync else '1')) == '1'
+    dry_run_mode = (os.getenv(DRY_RUN_ENV) or ('0' if args.no_dry_run else '1')) == '1'
+    cleanup_mode = (os.getenv(CLEANUP_ENV) or ('0' if args.no_cleanup else '1')) == '1'
```

Comment:
Looks good, we should document that in the README though.
Its fine in a follow up PR.

I also think we should accept more truthy values e.g. "true", "True", "TRUE", "y", "yes", ... Not sure if there is an accepted standard in k8s for this.

Reply:
:+1: I can fix this in future PRs